### PR TITLE
Fix empty result if bad column in the ORDER BY

### PR DESCRIPTION
**Pull request by michaljurecko**
Changes:
- Fixed bug with empty result set -> edge case.

### CODE REVIEW

File: src/Extractor/NodeJs/lib/Extractor.js [hunk 1]
```diff
@@ -67,6 +67,11 @@ class Extractor {
   }
 
   async processPage(page, pageIndex, resolve) {
+    if (!page.resources) {
```
Comment (michaljurecko) [inline src/Extractor/NodeJs/lib/Extractor.js:70]: Pri testovani tvojho PR som narazil na jednu chybu, ktora tam uz bola ... 

Ak nie je nacitany ziaden vysledok, tak `page.resources` je `[]` ... no pokial je v query nejaky key, ktory neexistuje v ziadnom dokumente (cosmosdb nema pevnu schemu) ... tak `page.resources` je `undefined`. Fixujem to a pridal som test ...

File: tests/functional/raw-no-data-bad-column/source/data/config.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "parameters": {
+    "db": {
+      "endpoint": "%env(string:ENDPOINT)%",
+      "#key": "%env(string:KEY)%",
+      "databaseId": "%env(string:DATABASE_ID)%"
+    },
+    "containerId": "simple",
+    "query": "SELECT * FROM c ORDER BY c.foo.bar.invalid",
```
Comment (michaljurecko) [inline tests/functional/raw-no-data-bad-column/source/data/config.json:9]: Kluc `c.foo.bar.invalid` sa nenachadza v ziadnom dokumente v kontajnery `simple` ... ale container `simple` nie je prazdny (edge case ked to padalo).

**Review by ondrajodas — APPROVED** [review]
jj super, taky jsem už na to narazil 👍
